2021-06-25 21:26:00,072 INFO [n-2cdc4b32dd916.a-2cdc479839e77.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:26:00,074 INFO [n-2cdc4b32dd916.a-2cdc479839e77.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 21:26:02,076 INFO [n-2cdc4b32dd916.a-2cdc479839e77.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 21:26:02,076 INFO [n-2cdc4b32dd916.a-2cdc479839e77.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:26:02,077 INFO [n-2cdc4b32dd916.a-2cdc479839e77.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 21:26:05,736 ERROR [n-2cdc4b32dd916.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:13,221 ERROR [n-2cdc4b32dd916.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:20,724 ERROR [n-2cdc4b32dd916.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:28,219 ERROR [n-2cdc4b32dd916.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:817)
	at java.base/sun.nio.ch.DatagramChannelImpl.blockingSend(DatagramChannelImpl.java:849)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.send(DatagramSocketAdaptor.java:217)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:531)
	at de.dailab.jiactng.agentcore.management.jmx.JmxConnectorManager.run(JmxConnectorManager.java:152)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
2021-06-25 21:26:04,062 ERROR [n-2cdc6363b4562.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-25 21:26:04,275 INFO [n-2cdc6363b4562.a-2cdc6420e1c8a.BidderBean] - Starting BidderBean.
2021-06-25 21:26:11,373 ERROR [n-2cdc6363b4562.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:18,894 ERROR [n-2cdc6363b4562.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:26,395 ERROR [n-2cdc6363b4562.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:30,315 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:58473 failed: java.net.SocketException: Connection reset
2021-06-25 21:26:30,317 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cdc6363b4562StandardJMSBroker#4 and tcp:///192.168.178.32:58461@58474 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 21:26:33,877 ERROR [n-2cdc6363b4562.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:37,192 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to send multicast message on interface lo
java.net.SocketException: Network is unreachable: no further information
	at java.base/sun.nio.ch.DatagramChannelImpl.send0(Native Method)
	at java.base/sun.nio.ch.DatagramChannelImpl.sendFromNativeBuffer(DatagramChannelImpl.java:897)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:859)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:817)
	at java.base/sun.nio.ch.DatagramChannelImpl.blockingSend(DatagramChannelImpl.java:849)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.send(DatagramSocketAdaptor.java:217)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:531)
	at de.dailab.jiactng.agentcore.management.jmx.JmxConnectorManager.run(JmxConnectorManager.java:152)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
2021-06-25 21:26:37,284 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-25 21:26:37,470 INFO [n-2cdcdf2efc41a.a-2cdcdfef46492.BidderBean] - Starting BidderBean.
2021-06-25 21:26:42,140 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to send multicast message on interface lo
java.net.SocketException: Network is unreachable: no further information
	at java.base/sun.nio.ch.DatagramChannelImpl.send0(Native Method)
	at java.base/sun.nio.ch.DatagramChannelImpl.sendFromNativeBuffer(DatagramChannelImpl.java:897)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:859)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:817)
	at java.base/sun.nio.ch.DatagramChannelImpl.blockingSend(DatagramChannelImpl.java:849)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.send(DatagramSocketAdaptor.java:217)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:531)
	at de.dailab.jiactng.agentcore.management.jmx.JmxConnectorManager.run(JmxConnectorManager.java:152)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
2021-06-25 21:26:42,243 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-25 21:26:42,444 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 21:26:42,445 WARN [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 21:26:43,861 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 21:26:43,861 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:26:43,865 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 21:26:44,598 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:45,872 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 21:26:45,873 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:26:45,876 INFO [n-2cdcee325d75e.a-2cdceab0dee1b.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 21:26:49,519 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:52,101 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:57,019 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:26:59,600 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:04,525 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:07,096 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:12,020 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:14,603 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:19,528 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:22,100 ERROR [n-2cdcdf2efc41a.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:25,834 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:58489 failed: java.net.SocketException: Connection reset
2021-06-25 21:27:25,838 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cdcee325d75eStandardJMSBroker#6 and tcp:///192.168.178.32:58479@58490 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 21:27:27,024 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:27:34,525 ERROR [n-2cdcee325d75e.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:28:17,602 ERROR [n-2cde554f23082.JmxConnectorManager] - Unable to send multicast message on interface lo
java.net.SocketException: Network is unreachable: no further information
	at java.base/sun.nio.ch.DatagramChannelImpl.send0(Native Method)
	at java.base/sun.nio.ch.DatagramChannelImpl.sendFromNativeBuffer(DatagramChannelImpl.java:897)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:859)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:817)
	at java.base/sun.nio.ch.DatagramChannelImpl.blockingSend(DatagramChannelImpl.java:849)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.send(DatagramSocketAdaptor.java:217)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:531)
	at de.dailab.jiactng.agentcore.management.jmx.JmxConnectorManager.run(JmxConnectorManager.java:152)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
2021-06-25 21:28:17,687 ERROR [n-2cde554f23082.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-25 21:28:17,898 INFO [n-2cde554f23082.a-2cde560010fb2.BidderBean] - Starting BidderBean.
2021-06-25 21:28:25,012 ERROR [n-2cde554f23082.JmxConnectorManager] - Unable to join multicast group 226.6.6.7
2021-06-25 21:49:02,889 WARN [org.springframework.context.support.ClassPathXmlApplicationContext] - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderNode' defined in class path resource [bidder.xml]: Cannot resolve reference to bean 'BidderAgent' while setting bean property 'agents' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderAgent' defined in class path resource [bidder.xml]: Cannot create inner bean 'BidderBean' of type [de.dailab.jiactng.aot.auction.client.BidderBean] while setting bean property 'agentBeans' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderBean' defined in class path resource [bidder.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'bidderId' of bean class [de.dailab.jiactng.aot.auction.client.BidderBean]: Bean property 'bidderId' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
2021-06-25 21:50:31,763 WARN [org.springframework.context.support.ClassPathXmlApplicationContext] - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderNode' defined in class path resource [bidder.xml]: Cannot resolve reference to bean 'BidderAgent' while setting bean property 'agents' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderAgent' defined in class path resource [bidder.xml]: Cannot create inner bean 'BidderBean' of type [de.dailab.jiactng.aot.auction.client.BidderBean] while setting bean property 'agentBeans' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderBean' defined in class path resource [bidder.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'groupToken' of bean class [de.dailab.jiactng.aot.auction.client.BidderBean]: Bean property 'groupToken' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
2021-06-25 21:50:49,072 WARN [org.springframework.context.support.ClassPathXmlApplicationContext] - Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderNode' defined in class path resource [bidder.xml]: Cannot resolve reference to bean 'BidderAgent' while setting bean property 'agents' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderAgent' defined in class path resource [bidder.xml]: Cannot create inner bean 'BidderBean' of type [de.dailab.jiactng.aot.auction.client.BidderBean] while setting bean property 'agentBeans' with key [0]; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'BidderBean' defined in class path resource [bidder.xml]: Error setting property values; nested exception is org.springframework.beans.NotWritablePropertyException: Invalid property 'messageGroup' of bean class [de.dailab.jiactng.aot.auction.client.BidderBean]: Bean property 'messageGroup' is not writable or has an invalid setter method. Does the parameter type of the setter match the return type of the getter?
2021-06-25 21:51:02,891 ERROR [n-2cf232de36d32.JmxConnectorManager] - Unable to send multicast message on interface lo
java.net.SocketException: Network is unreachable: no further information
	at java.base/sun.nio.ch.DatagramChannelImpl.send0(Native Method)
	at java.base/sun.nio.ch.DatagramChannelImpl.sendFromNativeBuffer(DatagramChannelImpl.java:897)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:859)
	at java.base/sun.nio.ch.DatagramChannelImpl.send(DatagramChannelImpl.java:817)
	at java.base/sun.nio.ch.DatagramChannelImpl.blockingSend(DatagramChannelImpl.java:849)
	at java.base/sun.nio.ch.DatagramSocketAdaptor.send(DatagramSocketAdaptor.java:217)
	at java.base/java.net.DatagramSocket.send(DatagramSocket.java:531)
	at de.dailab.jiactng.agentcore.management.jmx.JmxConnectorManager.run(JmxConnectorManager.java:152)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
2021-06-25 21:51:03,042 ERROR [n-2cf232de36d32.JmxConnectorManager] - Unable to leave multicast group 226.6.6.7
2021-06-25 21:51:03,279 INFO [n-2cf232de36d32.a-2cf233bfc9cd2.BidderBean] - Starting BidderBean.
2021-06-25 21:58:35,727 INFO [n-2cf8c663ed676.a-2cf8c6014f851.BidderBean] - Starting BidderBean.
2021-06-25 21:58:47,790 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 21:58:47,794 WARN [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 21:58:48,590 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 21:58:48,592 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:58:48,611 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 21:58:50,592 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 21:58:50,592 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:58:50,606 INFO [n-2cf8f036741c6.a-2cf8ef5382799.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 21:58:55,435 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:57953 failed: java.net.SocketException: Connection reset
2021-06-25 21:58:55,442 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cf8f036741c6StandardJMSBroker#2 and tcp:///192.168.178.32:63386@57954 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 21:59:00,603 INFO [n-2cf9249c3e80a.a-2cf9242e6b039.BidderBean] - Starting BidderBean.
2021-06-25 21:59:17,596 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:57960 failed: java.net.SocketException: Connection reset
2021-06-25 21:59:17,601 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cf9249c3e80aStandardJMSBroker#4 and tcp:///192.168.178.32:57951@57961 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 21:59:22,590 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 21:59:22,594 WARN [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 21:59:23,840 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 21:59:23,843 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:59:23,850 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 21:59:25,845 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 21:59:25,846 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 21:59:25,849 INFO [n-2cf9739305ede.a-2cf972b30d1bd.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 21:59:40,481 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:57967 failed: java.net.SocketException: Connection reset
2021-06-25 21:59:40,486 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cf9739305edeStandardJMSBroker#6 and tcp:///192.168.178.32:57958@57968 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 21:59:45,124 INFO [n-2cf9ca45ac2ca.a-2cf9c9d5e54bd.BidderBean] - Starting BidderBean.
2021-06-25 22:00:09,361 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2cf9739305edeStandardJMSBroker#8 and tcp:///192.168.178.32:64344@64346 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:00:09,361 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:64347 failed: java.net.SocketException: Connection reset
2021-06-25 22:00:10,588 WARN [n-2cf9739305ede.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:00:10,590 WARN [n-2cf9739305ede.IDirectory] - 	df@n-2cf9249c3e80a
2021-06-25 22:00:22,216 INFO [n-2cf9739305ede.a-2cf9715d2a108.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:00:22,220 ERROR [n-2cf9739305ede.a-2cf9715d2a108.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:00:40,589 WARN [n-2cf9739305ede.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:00:40,590 WARN [n-2cf9739305ede.IDirectory] - 	df@n-2cf9ca45ac2ca
2021-06-25 22:07:14,029 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:07:14,033 WARN [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:07:15,250 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:07:15,252 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:07:15,256 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:07:17,259 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:07:17,259 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:07:17,260 INFO [n-2d004fac98d1e.a-2d004ece9d2dd.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:07:18,563 INFO [n-2d00639057bae.a-2d00633001001.BidderBean] - Starting BidderBean.
2021-06-25 22:07:50,361 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:50196 failed: java.net.SocketException: Connection reset
2021-06-25 22:07:50,368 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d00639057baeStandardJMSBroker#4 and tcp:///192.168.178.32:50189@50197 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:13:36,113 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:13:36,117 WARN [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:13:37,392 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:13:37,394 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:13:37,402 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:13:39,397 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:13:39,398 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:13:39,400 INFO [n-2d05df438b3ae.a-2d05de6650d39.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:13:57,613 INFO [n-2d0632a91c372.a-2d06323d1fa01.BidderBean] - Starting BidderBean.
2021-06-25 22:14:35,770 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:14:35,774 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:15:35,673 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:15:35,675 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:16:35,650 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:16:35,651 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:17:35,691 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:17:35,693 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:18:35,674 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:18:35,676 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:19:06,414 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:62120 failed: java.net.SocketException: Connection reset
2021-06-25 22:19:06,423 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d05df438b3aeStandardJMSBroker#6 and tcp:///192.168.178.32:62117@62119 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:19:15,538 ERROR [n-2d0ad23bc3db2.a-2d0ad1e60d895] - Agentbean: 'BidderBean' could not switch to state: 'STARTED'! 
  Exception was: 
de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:149)
	at de.dailab.jiactng.agentcore.Agent.setBeanState(Agent.java:686)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:501)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStart(SimpleAgentNode.java:939)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.afterPropertiesSet(SimpleAgentNode.java:706)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:761)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.startAgentNode(SimpleAgentNode.java:198)
	at de.dailab.jiactng.aot.auction.StartBidder.main(StartBidder.java:12)
Caused by: java.net.UnknownHostException: Der angegebene Host ist unbekannt (266.6.6.7)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:932)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1517)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:851)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1507)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1366)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1300)
	at java.base/java.net.InetAddress.getByName(InetAddress.java:1250)
	at de.dailab.jiactng.aot.auction.client.BidderBean.doStart(BidderBean.java:92)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	... 21 more
2021-06-25 22:19:15,543 ERROR [n-2d0ad23bc3db2.a-2d0ad1e60d895] - Recovery for Agentbean: 'BidderBean' failed, removing Bean.
java.lang.RuntimeException: de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.AbstractAgentBean.handleLifecycleException(AbstractAgentBean.java:354)
	at de.dailab.jiactng.agentcore.Agent.handleBeanException(Agent.java:604)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:503)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStart(SimpleAgentNode.java:939)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.afterPropertiesSet(SimpleAgentNode.java:706)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:761)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.startAgentNode(SimpleAgentNode.java:198)
	at de.dailab.jiactng.aot.auction.StartBidder.main(StartBidder.java:12)
Caused by: de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:149)
	at de.dailab.jiactng.agentcore.Agent.setBeanState(Agent.java:686)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:501)
	... 19 more
Caused by: java.net.UnknownHostException: Der angegebene Host ist unbekannt (266.6.6.7)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:932)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1517)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:851)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1507)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1366)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1300)
	at java.base/java.net.InetAddress.getByName(InetAddress.java:1250)
	at de.dailab.jiactng.aot.auction.client.BidderBean.doStart(BidderBean.java:92)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	... 21 more
2021-06-25 22:19:35,673 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:19:35,674 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:19:39,119 WARN [n-2d05df438b3ae.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:19:39,121 WARN [n-2d05df438b3ae.IDirectory] - 	df@n-2d0632a91c372
2021-06-25 22:19:44,076 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d05df438b3aeStandardJMSBroker#8 and tcp:///192.168.178.32:53060@53062 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:19:44,076 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:53063 failed: java.net.SocketException: Connection reset
2021-06-25 22:19:51,770 ERROR [n-2d0b59702511e.a-2d0b591074ac5] - Agentbean: 'BidderBean' could not switch to state: 'STARTED'! 
  Exception was: 
de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:149)
	at de.dailab.jiactng.agentcore.Agent.setBeanState(Agent.java:686)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:501)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStart(SimpleAgentNode.java:939)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.afterPropertiesSet(SimpleAgentNode.java:706)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:761)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.startAgentNode(SimpleAgentNode.java:198)
	at de.dailab.jiactng.aot.auction.StartBidder.main(StartBidder.java:12)
Caused by: java.net.UnknownHostException: Der angegebene Host ist unbekannt (266.6.6.7)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:932)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1517)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:851)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1507)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1366)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1300)
	at java.base/java.net.InetAddress.getByName(InetAddress.java:1250)
	at de.dailab.jiactng.aot.auction.client.BidderBean.doStart(BidderBean.java:92)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	... 21 more
2021-06-25 22:19:51,775 ERROR [n-2d0b59702511e.a-2d0b591074ac5] - Recovery for Agentbean: 'BidderBean' failed, removing Bean.
java.lang.RuntimeException: de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.AbstractAgentBean.handleLifecycleException(AbstractAgentBean.java:354)
	at de.dailab.jiactng.agentcore.Agent.handleBeanException(Agent.java:604)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:503)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStart(SimpleAgentNode.java:939)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.afterPropertiesSet(SimpleAgentNode.java:706)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1687)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1624)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:555)
	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
	at org.springframework.beans.factory.support.DefaultListableBeanFactory.preInstantiateSingletons(DefaultListableBeanFactory.java:761)
	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:867)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:543)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:139)
	at org.springframework.context.support.ClassPathXmlApplicationContext.<init>(ClassPathXmlApplicationContext.java:83)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.startAgentNode(SimpleAgentNode.java:198)
	at de.dailab.jiactng.aot.auction.StartBidder.main(StartBidder.java:12)
Caused by: de.dailab.jiactng.agentcore.lifecycle.LifecycleException: Failed to start
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:149)
	at de.dailab.jiactng.agentcore.Agent.setBeanState(Agent.java:686)
	at de.dailab.jiactng.agentcore.Agent.doStart(Agent.java:501)
	... 19 more
Caused by: java.net.UnknownHostException: Der angegebene Host ist unbekannt (266.6.6.7)
	at java.base/java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method)
	at java.base/java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:932)
	at java.base/java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1517)
	at java.base/java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:851)
	at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1507)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1366)
	at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1300)
	at java.base/java.net.InetAddress.getByName(InetAddress.java:1250)
	at de.dailab.jiactng.aot.auction.client.BidderBean.doStart(BidderBean.java:92)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.start(AbstractLifecycle.java:139)
	... 21 more
2021-06-25 22:19:54,411 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d05df438b3aeStandardJMSBroker#10 and tcp:///192.168.178.32:64627@64629 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:19:54,411 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:64630 failed: java.net.SocketException: Connection reset
2021-06-25 22:20:09,120 WARN [n-2d05df438b3ae.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:20:09,121 WARN [n-2d05df438b3ae.IDirectory] - 	df@n-2d0ad23bc3db2
2021-06-25 22:20:35,680 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:20:35,681 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:21:35,674 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:21:35,675 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:22:35,676 INFO [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:22:35,676 ERROR [n-2d05df438b3ae.a-2d05dd0d4a4a8.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:22:52,461 INFO [n-2d0dfa7744bae.a-2d0dfa1766f29.BidderBean] - Starting BidderBean.
2021-06-25 22:22:57,809 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:52115 failed: java.net.SocketException: Connection reset
2021-06-25 22:22:57,818 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0dfa7744baeStandardJMSBroker#4 and tcp:///192.168.178.32:53573@52116 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:23:03,274 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:23:03,281 WARN [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:23:04,441 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:23:04,444 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:04,466 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:23:06,448 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:23:06,449 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:06,452 INFO [n-2d0e1fb8b7d1a.a-2d0e1ed34142d.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:23:10,619 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0dfa7744baeStandardJMSBroker#6 and tcp:///192.168.178.32:52120@52122 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:23:10,619 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:52123 failed: java.net.SocketException: Connection reset
2021-06-25 22:23:15,371 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:23:15,374 WARN [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:23:16,589 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:23:16,591 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:16,599 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:23:18,594 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:23:18,595 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:18,598 INFO [n-2d0e4cf97a352.a-2d0e4c1e837dd.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:23:20,664 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:56258 failed: java.net.SocketException: Connection reset
2021-06-25 22:23:20,669 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0e4cf97a352StandardJMSBroker#6 and tcp:///192.168.178.32:52113@56259 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:23:25,174 INFO [n-2d0e74708aa6e.a-2d0e740db8921.BidderBean] - Starting BidderBean.
2021-06-25 22:23:36,976 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0e4cf97a352StandardJMSBroker#8 and tcp:///192.168.178.32:56263@56265 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:23:36,977 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:56266 failed: java.net.SocketException: Connection reset
2021-06-25 22:23:45,736 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:23:45,742 WARN [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:23:46,850 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:23:46,852 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:46,858 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:23:48,850 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:23:48,852 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:23:48,855 INFO [n-2d0ebdb25b446.a-2d0ebcc973e61.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:23:52,750 INFO [n-2d0edb72df88e.a-2d0edb0ae2875.BidderBean] - Starting BidderBean.
2021-06-25 22:24:06,277 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:55517 failed: java.net.SocketException: Connection reset
2021-06-25 22:24:06,284 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0edb72df88eStandardJMSBroker#4 and tcp:///192.168.178.32:65309@55518 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:24:10,894 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:24:10,899 WARN [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:24:12,097 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:24:12,100 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:24:12,108 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:24:14,106 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:24:14,107 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:24:14,110 INFO [n-2d0f1bc7c2a56.a-2d0f1add13bc5.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:24:31,806 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:55525 failed: java.net.SocketException: Connection reset
2021-06-25 22:24:31,813 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0f1bc7c2a56StandardJMSBroker#6 and tcp:///192.168.178.32:55515@55526 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:25:10,452 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:25:10,454 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:25:13,869 WARN [n-2d0f1bc7c2a56.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:25:13,871 WARN [n-2d0f1bc7c2a56.IDirectory] - 	df@n-2d0edb72df88e
2021-06-25 22:25:22,057 INFO [n-2d1027e8bfa2a.a-2d102785f2da1.BidderBean] - Starting BidderBean.
2021-06-25 22:25:30,149 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d0f1bc7c2a56StandardJMSBroker#8 and tcp:///192.168.178.32:55773@55775 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:25:30,149 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:55776 failed: java.net.SocketException: Connection reset
2021-06-25 22:26:10,391 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:26:10,392 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:26:13,873 WARN [n-2d0f1bc7c2a56.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:26:13,874 WARN [n-2d0f1bc7c2a56.IDirectory] - 	df@n-2d1027e8bfa2a
2021-06-25 22:27:10,385 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:27:10,387 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:28:10,383 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:28:10,384 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:29:10,387 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:29:10,388 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:30:10,405 INFO [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - Writing result to C:\Users\hsams\IdeaProjects\aot34branch\logs\auction_0.txt
2021-06-25 22:30:10,407 ERROR [n-2d0f1bc7c2a56.a-2d0f198ec0f88.AuctionRunnerBean] - java.nio.file.NoSuchFileException: logs\auction_0.txt
2021-06-25 22:30:51,736 INFO [n-2d14f3e8401aa.a-2d14f38bffbc9.BidderBean] - Starting BidderBean.
2021-06-25 22:30:57,276 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:30:57,279 WARN [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:30:58,481 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:30:58,483 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:30:58,490 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:31:00,490 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:31:00,491 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:31:00,493 INFO [n-2d1505a47677e.a-2d1504c501dfd.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:31:28,061 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:60314 failed: java.net.SocketException: Connection reset
2021-06-25 22:31:28,064 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d14f3e8401aaStandardJMSBroker#4 and tcp:///192.168.178.32:60311@60313 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:31:54,744 WARN [n-2d14f3e8401aa.IDirectory] - Forget nodes due to missing alive message:

2021-06-25 22:31:54,745 WARN [n-2d14f3e8401aa.IDirectory] - 	df@n-2d1505a47677e
2021-06-25 22:33:59,076 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:33:59,083 WARN [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:34:00,278 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:34:00,282 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:34:00,289 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:34:02,285 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:34:02,286 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:34:02,289 INFO [n-2d17aae6be17a.a-2d17aa092d68d.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:34:05,239 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:52058 failed: java.net.SocketException: Connection reset
2021-06-25 22:34:05,245 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d17aae6be17aStandardJMSBroker#6 and tcp:///192.168.178.32:60306@52059 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:34:10,662 INFO [n-2d17d90e6ac8a.a-2d17d8ad734fd.BidderBean] - Starting BidderBean.
2021-06-25 22:34:17,077 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:52065 failed: java.net.SocketException: Connection reset
2021-06-25 22:34:17,082 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d17d90e6ac8aStandardJMSBroker#4 and tcp:///192.168.178.32:52056@52066 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:35:23,113 INFO [n-2d18a62e4f9bf.a-2d18a372dad87.BidderBean] - Starting BidderBean.
2021-06-25 22:35:30,466 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-2d18a62e4f9bfStandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-2d18a62e4f9bfStandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:35:30,469 ERROR [n-2d18a62e4f9bf.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:35:30,472 ERROR [n-2d18a62e4f9bf.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-2d18a62e4f9bf;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-2d18a62e4f9bf}, Payload: null, Sender: msgbox:df@n-2d18a62e4f9bf]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:35:30,481 WARN [n-2d18a62e4f9bf.a-2d18a372dad87.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:35:30,486 WARN [n-2d18a62e4f9bf.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d18a62e4f9bfStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:37:42,193 WARN [n-2d1914668a2cb.IDirectory] - Measured interval of sending alive message: 123211ms instead of 15000ms
2021-06-25 22:38:10,886 INFO [n-2d1914668a2cb.a-2d1911b496267.BidderBean] - Starting BidderBean.
2021-06-25 22:38:10,950 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-2d1914668a2cbStandardJMSBroker#1-4'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-2d1914668a2cbStandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:10,953 ERROR [n-2d1914668a2cb.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:10,955 ERROR [n-2d1914668a2cb.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-2d1914668a2cb;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-2d1914668a2cb}, Payload: null, Sender: msgbox:df@n-2d1914668a2cb]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:10,963 WARN [n-2d1914668a2cb.a-2d1911b496267.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d1914668a2cbStandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1914668a2cbStandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:10,966 WARN [n-2d1914668a2cb.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1914668a2cbStandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:39,840 INFO [n-2d1b6b2860a37.a-2d1b687f45a73.BidderBean] - Starting BidderBean.
2021-06-25 22:38:39,960 ERROR [org.apache.activemq.thread.TaskRunnerFactory] - Error in thread 'ActiveMQ VMTransport: vm://n-2d1b6b2860a37StandardJMSBroker#1-1'
org.apache.activemq.broker.BrokerStoppedException: Broker BrokerService[n-2d1b6b2860a37StandardJMSBroker] is being stopped
	at org.apache.activemq.broker.TransportConnection$1.onCommand(TransportConnection.java:193)
	at org.apache.activemq.transport.ResponseCorrelator.onCommand(ResponseCorrelator.java:116)
	at org.apache.activemq.transport.MutexTransport.onCommand(MutexTransport.java:50)
	at org.apache.activemq.transport.vm.VMTransport.iterate(VMTransport.java:270)
	at org.apache.activemq.thread.PooledTaskRunner.runTask(PooledTaskRunner.java:133)
	at org.apache.activemq.thread.PooledTaskRunner$1.run(PooledTaskRunner.java:48)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:39,965 ERROR [n-2d1b6b2860a37.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:72)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1412)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendAdvertisement(DirectoryAgentNodeBean.java:1529)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.access$200(DirectoryAgentNodeBean.java:73)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean$AgentNodePinger.run(DirectoryAgentNodeBean.java:1622)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:39,964 ERROR [n-2d1b6b2860a37.IDirectory.jms] - Sending of Message to address 'group:df@dfgroup' through JMS failed!
javax.jms.JMSException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:39,975 ERROR [n-2d1b6b2860a37.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-2d1b6b2860a37;JiacTNGProtocolID=>df:advertise;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-2d1b6b2860a37}, Payload: de.dailab.jiactng.agentcore.directory.Advertisement@390f66f3, Sender: msgbox:df@n-2d1b6b2860a37]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendAdvertisement(DirectoryAgentNodeBean.java:1529)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.access$200(DirectoryAgentNodeBean.java:73)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean$AgentNodePinger.run(DirectoryAgentNodeBean.java:1622)
	at java.base/java.util.TimerThread.mainLoop(Timer.java:556)
	at java.base/java.util.TimerThread.run(Timer.java:506)
Caused by: javax.jms.JMSException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:72)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1412)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 6 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:39,981 ERROR [n-2d1b6b2860a37.IDirectory] - sendMessage failed. Message:
[Headers: {JiacTNGSenderAddress=>msgbox:df@n-2d1b6b2860a37;JiacTNGProtocolID=>df:bye;JiacTNGGroupAddress=>df@dfgroup;UUID=>n-2d1b6b2860a37}, Payload: null, Sender: msgbox:df@n-2d1b6b2860a37]
de.dailab.jiactng.agentcore.comm.CommunicationException: error while sending message
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:221)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.sendMessage(DirectoryAgentNodeBean.java:1472)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doStop(DirectoryAgentNodeBean.java:295)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doStop(SimpleAgentNode.java:978)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.stop(AbstractLifecycle.java:181)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:819)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: javax.jms.JMSException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:54)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1398)
	at org.apache.activemq.ActiveMQConnection.syncSendPacket(ActiveMQConnection.java:1324)
	at org.apache.activemq.ActiveMQSession.send(ActiveMQSession.java:1971)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:288)
	at org.apache.activemq.ActiveMQMessageProducer.send(ActiveMQMessageProducer.java:223)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.sendMessage(JMSSender.java:52)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSSender.send(JMSSender.java:43)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.send(JMSMessageTransport.java:215)
	... 7 more
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:40,015 WARN [n-2d1b6b2860a37.a-2d1b687f45a73.CommunicationBean.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d1b6b2860a37StandardJMSBroker#3) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.comm.CommunicationBean.doCleanup(CommunicationBean.java:172)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.Agent.doCleanup(Agent.java:370)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:996)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#3) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:38:40,021 WARN [n-2d1b6b2860a37.IDirectory.jms] - could not stop JMS connection 
org.apache.activemq.ConnectionFailedException: The JMS connection has failed: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.ActiveMQConnection.checkClosedOrFailed(ActiveMQConnection.java:1471)
	at org.apache.activemq.ActiveMQConnection.doStop(ActiveMQConnection.java:575)
	at org.apache.activemq.ActiveMQConnection.stop(ActiveMQConnection.java:564)
	at de.dailab.jiactng.agentcore.comm.transport.jms.JMSMessageTransport.doCleanup(JMSMessageTransport.java:103)
	at de.dailab.jiactng.agentcore.directory.DirectoryAgentNodeBean.doCleanup(DirectoryAgentNodeBean.java:311)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.doCleanup(SimpleAgentNode.java:1011)
	at de.dailab.jiactng.agentcore.lifecycle.AbstractLifecycle.cleanup(AbstractLifecycle.java:221)
	at de.dailab.jiactng.agentcore.SimpleAgentNode.shutdown(SimpleAgentNode.java:820)
	at de.dailab.jiactng.agentcore.SimpleAgentNode$1.run(SimpleAgentNode.java:144)
Caused by: org.apache.activemq.transport.TransportDisposedIOException: peer (vm://n-2d1b6b2860a37StandardJMSBroker#1) stopped.
	at org.apache.activemq.transport.vm.VMTransport.stop(VMTransport.java:229)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.TransportFilter.stop(TransportFilter.java:65)
	at org.apache.activemq.transport.ResponseCorrelator.stop(ResponseCorrelator.java:132)
	at org.apache.activemq.broker.TransportConnection.doStop(TransportConnection.java:1164)
	at org.apache.activemq.broker.TransportConnection$4.run(TransportConnection.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:831)
2021-06-25 22:39:14,105 INFO [n-2d1c43bf6f706.a-2d1c435175f01.BidderBean] - Starting BidderBean.
2021-06-25 22:39:24,482 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:39:24,489 WARN [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:39:25,671 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:39:25,673 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:39:25,681 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:39:27,681 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:39:27,681 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:39:27,683 INFO [n-2d1c6722ec726.a-2d1c663931689.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:39:44,146 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:62989 failed: java.net.SocketException: Connection reset
2021-06-25 22:39:44,152 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d1c6722ec726StandardJMSBroker#6 and tcp:///192.168.178.32:62982@62990 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:39:48,557 INFO [n-2d1cc3c274bce.a-2d1cc34ef6fb1.BidderBean] - Starting BidderBean.
2021-06-25 22:39:56,212 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:63600 failed: java.net.SocketException: Connection reset
2021-06-25 22:39:56,217 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d1cc3c274bceStandardJMSBroker#4 and tcp:///192.168.178.32:62987@63601 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:40:05,074 INFO [n-2d1d00992b5fa.a-2d1d00450c801.BidderBean] - Starting BidderBean.
2021-06-25 22:40:09,382 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:40:09,386 WARN [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:40:10,618 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:40:10,620 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:40:10,628 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:40:12,619 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:40:12,621 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:40:12,622 INFO [n-2d1d0e8914f2e.a-2d1d0da92e511.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:40:27,775 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:63613 failed: java.net.SocketException: Connection reset
2021-06-25 22:40:27,779 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d1d00992b5faStandardJMSBroker#4 and tcp:///192.168.178.32:63610@63612 shutdown due to a remote error: java.net.SocketException: Connection reset
2021-06-25 22:40:32,748 INFO [n-2d1d65981a256.a-2d1d64bab5111.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:40:32,751 WARN [n-2d1d65981a256.a-2d1d64bab5111.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:40:33,995 INFO [n-2d1d65981a256.a-2d1d64bab5111.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:40:33,997 INFO [n-2d1d65981a256.a-2d1d64bab5111.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:40:34,005 INFO [n-2d1d65981a256.a-2d1d64bab5111.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:40:40,353 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Reading group tokens from file C:\Users\hsams\aotgrouptokens.txt
2021-06-25 22:40:40,364 WARN [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - File not found! Allowing any group token.
2021-06-25 22:40:41,417 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Current Phase: STARTING
2021-06-25 22:40:41,419 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:40:41,426 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Sending StartAuctions(1, aot-test-auction) to group
2021-06-25 22:40:43,422 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Current Phase: REGISTRATION
2021-06-25 22:40:43,423 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - Messages in Memory: 0
2021-06-25 22:40:43,426 INFO [n-2d1d814d64e2a.a-2d1d806b85845.AuctioneerMetaBean] - No one showed up for the auction. :-(
2021-06-25 22:40:48,072 INFO [n-2d1da1a2ad8ca.a-2d1da1372e9e5.BidderBean] - Starting BidderBean.
2021-06-25 22:41:18,918 WARN [org.apache.activemq.broker.TransportConnection.Transport] - Transport Connection to: tcp://192.168.178.32:55880 failed: java.net.SocketException: Connection reset
2021-06-25 22:41:18,920 WARN [org.apache.activemq.network.DemandForwardingBridgeSupport] - Network connection between vm://n-2d1d814d64e2aStandardJMSBroker#6 and tcp:///192.168.178.32:55877@55879 shutdown due to a remote error: java.net.SocketException: Connection reset
